x/inter-tx/ibc_module.go

```diff
@@ -0,0 +1,130 @@
+package inter_tx
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	capabilitytypes "github.com/cosmos/cosmos-sdk/x/capability/types"
+
+	"github.com/cosmos/ibc-go/v2/modules/apps/27-interchain-accounts/controller/keeper"
+	channeltypes "github.com/cosmos/ibc-go/v2/modules/core/04-channel/types"
+	ibcexported "github.com/cosmos/ibc-go/v2/modules/core/exported"
+)
+
+// IBCModule implements the ICS26 interface for interchain accounts controller chains
+type IBCModule struct {
+	keeper keeper.Keeper
```

Comment:
I think this needs to be the interTx keeper instead of the ica controller keeper; so that the scopedKeeper in interTx can claim the channel capability passed back in the `OnChanOpenInit` callback.